README.md

```diff
@@ -1,7 +1,5 @@
 # Linphone on iPhone
 
-[![Build Status](https://travis-ci.org/BelledonneCommunications/linphone-iphone.svg?branch=master)](https://travis-ci.org/BelledonneCommunications/linphone-iphone)
```

Comment:
On second thought, we'll deal with this conflict when we go to merge this back upstream later with linphone-iphone proper.